### ISSUE TITLE
Download of v0.0.3 is blocked by Microsoft Edge - "Couldn't download - Virus detected"

### ISSUE THREAD

**Issue by ok-royal** [Lyall/DAFix#2]
Probably a false positive but I thought I'd let you know. I'm not getting this error when downloading v0.8.5 of the FFXVIFix for example.

**Environment:**
Microsoft Edge v131.0.2903.70 on Windows 11

**Relevant attachments:**
* [DAFix_v0.0.3.zip](https://github.com/Lyall/DAFix/releases/download/v0.0.3/DAFix_v0.0.3.zip) - a link to download the file from the Releases page
* virus-report.png - an image displaying the issue according to Windows Defender

![virus-report](https://github.com/user-attachments/assets/abdf618b-ef51-4c1c-a7ae-6b600c1babdd)

**Comment by Lyall** [comment]
Yeah this is another false positive label by Microsoft Defender. I'll fill out their submission form to try and get the false positive label removed (though I'm not sure if they are even read by humans). 

For now you'll have to allow it through Microsoft Defender or wait and see until/if a definition update clears the label.

**Comment by rainichan** [comment]
Figured you should know that VirusTotal is still reporting this as multiple trojans as well - 
![virustotal](https://github.com/user-attachments/assets/8b112e30-6307-4be6-98d5-d577edf27fe6)

current VT link: https://www.virustotal.com/gui/file/613a3d005da6d95ed773b3281e1d8518c71d91effd1ae32e27d6a5986b64059a

Chrome seems to be letting it through now though where it wasn't before and flagging it as a virus, it's just Windows Defender throwing a fit on my end.
